# 0x05. Python - Exceptions
  
## A project on:
- The difference between errors and exceptions
- What are exceptions and how to use them
- How to raise a builtin exception
- When to implement a clean-up action after an exception

## Requirements
- All  files are interpreted/compiled on Ubuntu 14.04 LTS using python3
- Code follows PEP 8 style

## Tasks

**[0-safe_print_list.py](0-safe_print_list.py)** - A function that prints x elements in a list

**[1-safe_print_integer.py](1-safe_print_integer.py)** - A function that prints an integer with "{:d}".format()

**[2-safe_print_list_integers.py](2-2-safe_print_list_integers.py)** - A function that prints the first x elements of a list and only integers.

**[3-safe_print_division.py](3-safe_print_division.py)** - A function that divides 2 integers and prints the result

**[4-list_division.py](4-list_division.py)** - A function that divides element by element 2 lists

**[5-raise_exception.py](5-raise_exception.py)** - A function that raises a type exception

**[6-raise_exception_msg.py](6-raise_exception_msg.py)** - A function that raises a name exception with a message
